packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -2487,75 +2442,50 @@ export class LitNodeClientNodeJs
     let siwe_statement = 'Lit Protocol PKP session signature';
     if (params.statement) {
       siwe_statement += ' ' + params.statement;
+      log(`[signSessionKey] statement found in params: "${params.statement}"`);
     }
 
     let siweMessage;
 
-    if (params?.resourceAbilityRequests) {
-      const resources = params.resourceAbilityRequests.map((r) => r.resource);
-
-      const recapObject =
-        await this.generateSessionCapabilityObjectWithWildcards(resources);
-
-      params.resourceAbilityRequests.forEach((r) => {
-        recapObject.addCapabilityForResource(r.resource, r.ability);
-
-        const verified = recapObject.verifyCapabilitiesForResource(
-          r.resource,
-          r.ability
-        );
-
-        if (!verified) {
-          throw new Error('Failed to verify capabilities for resource');
-        }
-      });
+    const siweParams = {
+      domain: params?.domain || globalThis.location?.host || 'litprotocol.com',
+      walletAddress: pkpEthAddress,
+      statement: siwe_statement,
+      uri: sessionKeyUri,
+      version: '1',
+      chainId: params.chainId ?? 1,
+      expiration: _expiration,
+      nonce: this.latestBlockhash!,
+    };
 
-      // regular siwe
-      siweMessage = new siwe.SiweMessage({
-        domain:
-          params?.domain || globalThis.location?.host || 'litprotocol.com',
-        address: pkpEthAddress,
-        statement: siwe_statement,
-        uri: sessionKeyUri,
-        version: '1',
-        chainId: params.chainId ?? 1,
-        expirationTime: _expiration,
-        resources: params.resources,
-        nonce: this.latestBlockhash!,
+    if (params.resourceAbilityRequests) {
```

Comment:
SessionSigs should always have resources?

Reply:
don't think you need that on cayenne?

Reply:
Any updates @Ansonhkg ?

Reply:
This is fixed